defold-simpledata/editor/src/simpledata-ext.clj

```diff
@@ -132,22 +139,13 @@
   (inherits resource-node/ResourceNode)
 
   ;; Editable properties.
-  (property name g/Str
-            (default "untitled")
-            (dynamic error (g/fnk [_node-id name] (validate-name _node-id name))))
-  (property f32 g/Num
-            (default 0.0)
-            (dynamic error (g/fnk [_node-id f32] (validate-f32 _node-id f32))))
-  (property u32 g/Int
-            (default 0)
-            (dynamic error (g/fnk [_node-id u32] (validate-u32 _node-id u32))))
-  (property i32 g/Int
-            (default 0))
-  (property u64 g/Int
-            (default 0)
-            (dynamic error (g/fnk [_node-id u64] (validate-u64 _node-id u64))))
-  (property i64 g/Int
-            (default 0))
+  (property name g/Str (dynamic error (g/fnk [_node-id name] (validate-name _node-id name))))
+  (property f32 g/Num (dynamic error (g/fnk [_node-id f32] (validate-f32 _node-id f32))))
+  (property u32 g/Int (dynamic error (g/fnk [_node-id u32] (validate-u32 _node-id u32))))
+  (property i32 g/Int)
+  (property u64 g/Int (dynamic error (g/fnk [_node-id u64] (validate-u64 _node-id u64))))
+  (property i64 g/Int)
+  (property v3 types/Vec3)
```

Comment:
I removed the `default` assignments here, because these will always be overwritten by whatever is defined in the `template` file, and these were just confusing matters.